examples/pennylane_skip_plugins_hybrid.py

```diff
@@ -0,0 +1,391 @@
+# Now, let's use the :class:`~.GradientDescentOptimizer`, and update the circuit
+# parameters over 100 optimization steps.
+
+# initialise the optimizer
+opt = qml.GradientDescentOptimizer(stepsize=0.4)
+
+# set the number of steps
+steps = 100
+# set the initial parameter values
+params = init_params
+
+for i in range(steps):
+    # update the circuit parameters
+    params = opt.step(cost, params)
+
+    if (i + 1) % 5 == 0:
+        print("Cost after step {:5d}: {: .7f}".format(i + 1, cost(params)))
+
+print("Optimized rotation angles: {}".format(params))
+
+##############################################################################
+# .. rst-class:: sphx-glr-script-out
+#
+#  Out:
+#
+#  .. code-block:: none
+#
+#     Optimized rotation angles: [ 1.57079633  0.01      ]
+
+
+##############################################################################
+# Comparing this to the :ref:`exact calculation <photon_redirection_calc>` above,
+# this is close to the optimum value of :math:`\theta=\pi/2`, while the value of
+# :math:`\phi` has not changed — consistent with the fact that :math:`\braket{\hat{n}_1}`
+# is independent of :math:`\phi`.
+#
+# .. _hybrid_computation_example:
+#
+# Hybrid computation
+# ------------------
+#
+# To really highlight the capabilities of PennyLane, let's now combine the qubit-rotation QNode
+# from the :ref:`qubit rotation tutorial <qubit_rotation>` with the CV photon-redirection
+# QNode from above, as well as some classical processing, to produce a truly hybrid
+# computational model.
+#
+# First, we define a computation consisting of three steps: two quantum nodes (the qubit rotation
+# and photon redirection circuits, running on the ``'default.qubit'`` and
+# ``'strawberryfields.fock'`` devices, respectively), along with a classical function, that simply
+# returns the squared difference of its two inputs using NumPy:
+
+# create the devices
+dev_qubit = qml.device("default.qubit", wires=1)
+dev_fock = qml.device("strawberryfields.fock", wires=2, cutoff_dim=10)
+
+
+@qml.qnode(dev_qubit)
+def qubit_rotation(phi1, phi2):
+    """Qubit rotation QNode"""
+    qml.RX(phi1, wires=0)
+    qml.RY(phi2, wires=0)
+    return qml.expval(qml.PauliZ(0))
+
+
+@qml.qnode(dev_fock)
+def photon_redirection(params):
+    """The photon redirection QNode"""
+    qml.FockState(1, wires=0)
+    qml.Beamsplitter(params[0], params[1], wires=[0, 1])
+    return qml.expval(qml.MeanPhoton(1))
+
+
+def squared_difference(x, y):
+    """Classical node to compute the squared
+    difference between two inputs"""
+    return np.abs(x - y) ** 2
+
+
+##############################################################################
+# Now, we can define an objective function associated with the optimization, linking together
+# our three subcomponents. Here, we wish to
+# perform the following hybrid quantum-classical optimization:
+#
+# .. figure:: ../../examples/figures/hybrid_graph.png
```

Comment:
this figure could use some padding above and below

Reply:
This won't render in the notebooks, it'll show up as a raw `<br>` tag (we used to have it, and I removed it for this reason).

Should I still add it back in?

Reply:
can we add padding into the image itself?